src/github.rs

```diff
@@ -376,7 +376,7 @@ pub struct Issue {
     /// Note that this is constructed via the [`Issue::repository`] method.
     /// It is not deserialized from the GitHub API.
     #[serde(skip)]
-    pub repository: OnceCell<IssueRepository>,
+    pub repository: OnceLock<IssueRepository>,
```

Comment:
We should set `rust-version` in `Cargo.toml` now that we started using "recent" Rust features.

Reply:
Set to `1.80.0`.